### PR TITLE
Issue/4627 Replacing plurals with common strings

### LINKED ISSUE #4627: Migrate plural string resources to regular string resources
Since GlotPress doesn't support plurals yet, we need to migrate all the current `plurals` of the app to regular string resources, otherwise, they won't get translated.

### PR DESCRIPTION
**Pull request by kidinov**
### Replacing plurals with common strings

Closes: #4627 

### Description
GlotPress does not support plurals at the moment, so we can not use them. I replaced them. Also, added this fact into the doc.

One commit contains replacing of one plural

### Testing instructions

Run:
* Shipping label creation
* Order refund
* Product creation

flows and check the work of that changed string as expected with different quantities

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/ShippingLabelPackagesAdapter.kt [hunk 1]
```diff
@@ -127,11 +127,14 @@ class ShippingLabelPackagesAdapter(
             val uiModel = uiModels[position]
             val shippingLabelPackage = uiModel.data
             binding.packageName.text = shippingLabelPackage.getTitle(context)
-            binding.packageItemsCount.text = "- ${context.resources.getQuantityString(
-                R.plurals.shipping_label_package_details_items_count,
-                shippingLabelPackage.itemsCount,
-                shippingLabelPackage.itemsCount
+
+            binding.packageItemsCount.text = "- ${StringUtils.getQuantityString(
+                context = binding.packageItemsCount.context,
```
Comment (hichamboushaba) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/ShippingLabelPackagesAdapter.kt]: np, there is a `context` variable in scope, so you can shorten this to `context = context`

File: WooCommerce/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -497,10 +497,8 @@
     <string name="shipping_label_package_details_weight_hint">Total package weight (%1$s)</string>
     <string name="shipping_label_package_details_weight_info">Includes package weight</string>
     <string name="shipping_label_packages_loading_error">Unable to load package definitions</string>
-    <plurals name="shipping_label_package_details_items_count">
-        <item quantity="one">%d item</item>
-        <item quantity="other">%d items</item>
-    </plurals>
+    <string name="shipping_label_package_details_items_count_one">%d item</string>
+    <string name="shipping_label_package_details_items_count_other">%d items</string>
```
Comment (hichamboushaba) [inline WooCommerce/src/main/res/values/strings.xml]: np, I know you named those `other` to match the plural quantity, by IMHO suffixing with `multiple` or `many` will be easier to understand for the translators.

**Review by hichamboushaba — APPROVED** [review]
Thanks for working on this @kidinov, this looks good, I left two np comments, but feel free to ignore them if you don't agree with them 🙂

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/ShippingLabelPackagesAdapter.kt (hunk 1 above)
Comment (kidinov) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/ShippingLabelPackagesAdapter.kt]: Good point! Fixed - https://github.com/woocommerce/woocommerce-android/pull/4636/commits/836a661f6916f0df9acb70fe88676925169baad3

File: WooCommerce/src/main/res/values/strings.xml (hunk 2 above)
Comment (kidinov) [reply WooCommerce/src/main/res/values/strings.xml]: Agree. Changed there - https://github.com/woocommerce/woocommerce-android/pull/4636/commits/4824fa7cc078cf16676be21a36a32ac55d6f2827
